src/components/v5/common/CompletedAction/partials/SplitPayment/partials/SplitPaymentTable/hooks.tsx

```diff
@@ -0,0 +1,105 @@
+import { createColumnHelper } from '@tanstack/react-table';
+import { BigNumber } from 'ethers';
+import React, { useMemo } from 'react';
+
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import { ExpenditureStatus } from '~gql';
+import { useTablet } from '~hooks';
+import useWrapWithRef from '~hooks/useWrapWithRef.ts';
+import { formatText } from '~utils/intl.ts';
+import { getSelectedToken } from '~utils/tokens.ts';
+import SplitPaymentPayoutsTotal from '~v5/common/ActionSidebar/partials/forms/SplitPaymentForm/partials/SplitPaymentPayoutsTotal/SplitPaymentPayoutsTotal.tsx';
+import { calculatePercentageValue } from '~v5/common/ActionSidebar/partials/forms/SplitPaymentForm/partials/SplitPaymentRecipientsField/utils.ts';
+
+import AmountField from '../../../PaymentBuilder/partials/PaymentBuilderTable/partials/AmountField/AmountField.tsx';
+import RecipientField from '../../../PaymentBuilder/partials/PaymentBuilderTable/partials/RecipientField/RecipientField.tsx';
+
+import { type SplitPaymentTableModel } from './types.ts';
+
+export const useGetSplitPaymentColumns = (
+  data: SplitPaymentTableModel[],
+  status: ExpenditureStatus,
+  isLoading?: boolean,
+) => {
+  const dataRef = useWrapWithRef(data);
```

Comment:
Can we go away from using this hook? It's not actually doing anything apart from 'hiding' the value from React eslint rules so that it doesn't complain about it missing from `useMemo` deps

A much better option that makes the intentions clear would be to just disable the eslint rule, and leave a comment about the reasons.